langs/python.go

```diff
@@ -107,6 +108,7 @@ def handler(ctx, data: io.BytesIO=None):
     except (Exception, ValueError) as ex:
         print(str(ex))
```

Comment:
@srpurani - while you are here, can you fix this one as well? Either add Flush or use logger?

Reply:
Done.